### DISCUSSION TITLE
Flutter Support

### DISCUSSION THREAD

**Discussion by joneswe** [vercel/next.js#13154] [Help]
It would be nice to have flutter support in future
Or somehow a next for flutter

**Comment by timneutkens** [comment]
> Flutter is Google’s UI toolkit for building beautiful, natively compiled applications for mobile, web, and desktop from a single codebase.

> Flutter apps are written in the Dart language and make use of many of the language's more advanced features. (https://en.wikipedia.org/wiki/Flutter_(software))

Currently Next.js' main focus is providing a great framework for building web applications and websites and providing the best developer experience / user experience. It does so using React and JavaScript / TypeScript.

**Comment by donpablo1977** [comment]
Please, add support to flutter technology

**Reply by timneutkens** [reply]
Read this: https://github.com/vercel/next.js/discussions/13154#discussioncomment-17108

**Reply by donpablo1977** [reply]
So, we don’t have any chance? 😀

**Comment by sr229** [comment]
Recently Flutter added Web targeting. This should be the focus of Vercel instead of Next.js.

**Comment by themisir** [comment]
I deployed flutter example for [form-validator](https://github.com/TheMisir/form-validator) package using vercel. Here's configuration:

Build command:

```sh
cd example && if cd flutter; then git pull && cd ..; else git clone https://github.com/flutter/flutter.git; fi && flutter/bin/flutter config --enable-web && flutter/bin/flutter build web --release
```

Build output:

```
example/build/web
```

PS: `example` is project directory.
